jetbrains-gateway/resources/META-INF/ext-gateway.xml

```diff
@@ -8,7 +8,12 @@
     <extensions defaultExtensionNs="com.intellij">
         <applicationService serviceImplementation="software.aws.toolkits.jetbrains.gateway.connection.SshAgentService" />
         <applicationService serviceImplementation="software.aws.toolkits.jetbrains.gateway.connection.ThinClientTrackerService" />
+        <notificationGroup id="aws.toolkit_deprecation" displayType="BALLOON" key="aws.toolkit_deprecation.title" />
```

Comment:
hasn't this notification group been declared already?

Reply:
Changed names to avoid confusion, the notification here will have a different message hence creating a new group

Reply:
it's a different message, but the same category of content, so it's unclear why we need a new one

Reply:
Makes sense, added the new notification to the original group.